app/controllers/nonsipp/landorproperty/LandPropertyAddressManualController.scala

```diff
@@ -199,13 +199,19 @@ object LandPropertyAddressManualController {
       "landPropertyAddressManual.field5.error.max"
```

Comment:
I appreciate this wasn't implemented by you, but I noticed that the messages.en line this relates to is "Enter a postcode in the right format, like AB1 1AB", which seems incorrect. Perhaps something like "Postal code must be 35 characters or fewer" would be better?

Reply:
"Enter a postcode in the right format, like AB1 1AB" is how it is in the content document attached to the original story and it is a similar text for the validation message on the postcode lookup page. If you think it could be improved we may need to discuss with our content designer. For now I'd keep it.